opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java

```diff
@@ -321,7 +327,7 @@ protected String findOriginalName(String fullpath, String changeset)
             ABBREV_LOG,
             "--abbrev-commit",
             "--name-status",
-            "--pretty=format:commit %h%b",
+            "--pretty=format:commit %h%n%d",
```

Comment:
%h - commit hash
%n - new line
%d - file references (renames, add, remove)

Reply:
so this in fact does not change the output, or does it ?

Reply:
Not really.
%b is too verbose - body + files
%d is just the files

Reply:
I see.